packages/react/src/components/ItemLayout/ItemLayout.tsx

```diff
@@ -117,8 +129,8 @@ class ItemLayout extends UIComponent<ReactProps<ItemLayoutProps>, any> {
     renderHeaderArea: (props, state, classes) => {
       const { debug, header, headerMedia, truncateHeader, headerCSS, headerMediaCSS } = props
 
-      const mergedClasses = cx('ui-item-layout__header', classes.header)
-      const mediaClasses = cx('ui-item-layout__headerMedia', classes.headerMedia)
+      const mergedClasses = cx(ItemLayout.slotClassNames.header, classes.header)
```

Comment:
lets rename these variables to `headerClasses` and `headerMediaClasses` - although, surely, this is a problem that has existed long before this PR